clients/flash/air-client/src/org/bigbluebutton/air/common/views/NoTabView.as

```diff
@@ -20,12 +22,18 @@ package org.bigbluebutton.air.common.views {
 			_topToolbar = createToolbar();
 			_topToolbar.percentWidth = 100;
 			addElement(_topToolbar);
+			addEventListener(Event.ADDED_TO_STAGE, onAddedToStage, false, 0, true);
+		}
+		
+		protected function onAddedToStage(event:Event):void {
+			stage.addEventListener(StageOrientationEvent.ORIENTATION_CHANGE, onOrientationChange, false, 0, true);
 		}
 		
 		/**
 		 * Override this method in subclasses to be notified of rotation changes
 		 */
-		public function rotationHandler(rotation:String):void {
+		protected function onOrientationChange(event:Event):void {
+			invalidateDisplayList();
```

Comment:
Why do you invalidate the display list on rotation? The updateDisplayList() function should already be getting called because the View size changes when the stage size changes.